app/assets/stylesheets/_colors.sass

```diff
@@ -0,0 +1,7 @@
+// This is the RGSoC teams-app color stack:
+
+$main: 		#333
+$railsred:	#d82222
+$lightgray:	#ddd
+$darkgray: 	#999
+$white: 	#fff
```

Comment:
You're using tabs here for alignment, use spaces instead (tabs are not guaranteed to have an exact/equal width).